tests/native-lib/pass/scalar_arguments.rs

```diff
@@ -39,5 +39,54 @@ fn main() {
 
         // test void function that prints from C
         printer();
+
+        test_pass_struct();
+        test_pass_struct_complex();
+    }
+}
+
+/// Test passing a basic struct as an argument.
+fn test_pass_struct() {
```

Comment:
The file these tests are in is called "scalar_arguments". Do these arguments you are passing here ,ook like scalars to you? :)

Please make a new file for aggregate arguments.